gcn/notices/svom/trigger.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/trigger.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM trigger alerts",
+  "description": "ECLAIRs or GRM trigger parameters computed onboard",
+  "allOf": [
+    { "$ref": "../core/AdditionalInfo.schema.json" },
+    { "$ref": "../core/Alert.schema.json" },
+    { "$ref": "../core/DateTime.schema.json" },
+    { "$ref": "../core/Event.schema.json" },
+    { "$ref": "../core/Localization.schema.json" },
+    { "$ref": "../core/Reporter.schema.json" },
+    { "$ref": "../core/Statistics.schema.json" },
+    { "$ref": "../core/Pointing.schema.json" },
```

Comment:
Hi @timroland, could you please tell us where you have used the Pointing Schema in examples? I was going through the examples, and all of them have (ra, dec, ra_dec_error), these parameters together should be used for the Localization schema. Is at any place are you using Pointing Schema?